scm-webapp/src/main/java/sonia/scm/security/JwtAccessTokenBuilder.java

```diff
@@ -183,8 +182,8 @@ public JwtAccessToken build() {
 
 
     if (refreshableFor > 0) {
-      long refreshExpiration = refreshableForUnit.toMillis(refreshableFor);
-      claims.put(JwtAccessToken.REFRESHABLE_UNTIL_CLAIM_KEY, new Date(now.toEpochMilli() + refreshExpiration).getTime());
+      long re = refreshableForUnit.toMillis(refreshableFor);
```

Comment:
What was your issue with `refreshExpiration`?

Reply:
`refreshExpiration` hides a member variable and was reported by SonarQube